### ISSUE TITLE
Rank role matchmaking

### ISSUE THREAD

**Issue by EQNutchapol** [ValveSoftware/Dota2-Gameplay#8332]
### Description

Rank role now is randomly put you in either classic ranked all pick or ranked role.

The indication is that when you find the match as role ranking, you will see that the match you are finding is not ranked role.

### Example Match ID (and possibly Timestamp)

-

### Screenshots

https://user-images.githubusercontent.com/87842346/233776702-f68668d9-738f-4283-b222-dade0d8b0bbf.mp4

**Comment by findtravishere** [comment]
i do not think this is a bug other than a UI not reflecting that it's queueing for an immortal draft ranked game. immortal level matchmaking no longer allows for ranked roles due to the new drafting system.

**Comment by stellardropthebeat** [comment]
at first I think it's a new drafting system, but sometimes I still get it role matchmaking.
